jsonnet/kube-prometheus/platforms/kops.libsonnet

```diff
@@ -18,22 +18,22 @@ local service(name, namespace, labels, selector, ports) = {
     kubeControllerManagerPrometheusDiscoveryService: service(
       'kube-controller-manager-prometheus-discovery',
       'kube-system',
-      { 'app.kubernetes.io/name': 'kube-controller-manager' },
-      { 'app.kubernetes.io/name': 'kube-controller-manager' },
+      { 'k8s-app': 'kube-controller-manager' },
+      { 'k8s-app': 'kube-controller-manager' },
       [{ name: 'https-metrics', port: 10257, targetPort: 10257 }]
     ),
     kubeSchedulerPrometheusDiscoveryService: service(
       'kube-scheduler-prometheus-discovery',
       'kube-system',
-      { 'app.kubernetes.io/name': 'kube-scheduler' },
-      { 'app.kubernetes.io/name': 'kube-scheduler' },
+      { 'k8s-app': 'kube-scheduler' },
```

Comment:
Same as above

Reply:
@paulfantom Kept both labels and change base branch to `main`.

Reply:
See my comment about `kube-controller-manager` as it is applicable also here.